test/specs/commonTests.js

```diff
@@ -873,7 +873,7 @@ export const implementsTextAlignProp = (Component, alignments = SUI.TEXT_ALIGNME
     _noDefaultClassNameFromProp(Component, 'textAlign', options)
     _noClassNameFromBoolProps(Component, 'textAlign', options)
 
-    _.each(alignments, propVal => {
+    alignments.forEach(propVal => {
```

Comment:
Lodash's `each` will work even `alignments` will not be array, we need to be more strict there.